src/utils.ts

```diff
@@ -1,5 +1,13 @@
+import bignum from 'bignum';
+import { ConfigManager } from 'confmgr/lib';
 import log4js from 'log4js';
 
+import { faucetConfig } from './faucetConfig';
+
+const config = faucetConfig('server');
+const decimals = config.Get('BACKEND', 'NETWORK_DECIMALS') as number;
+const MAX_ALLOWED_DRIP_FLOAT = 9999999.9;
```

Comment:
Add an assertion `assert(Number.isSafeInteger(MAX_ALLOWED_DRIP_FLOAT))` for sanity's sake

Reply:
This suggestion actually doesn't work because Number.isSafeInteger doesn't work for floats, sorry